### PR TITLE
Fix SWIG wrapper for export_cash so that it returns two values.

### PR DESCRIPTION
**Pull request by weissjeffm**
This is @fellowtravler 's suggestion, tests run fine with this fix.

### CODE REVIEW

**Comment by lclc** [comment]
@monetas-ci re-test this please

File: wrappers/opentxs.i [hunk 1]
```diff
@@ -103,6 +103,7 @@ namespace std {
 
 
 %rename(OTRecordLessThan) opentxs::OTRecord::operator<(const OTRecord& rhs); 
+%apply std::string &OUTPUT { std::string & STR_RETAINED_COPY };
```
Comment (toxeus) [inline wrappers/opentxs.i]: Formatting problem:
`s/std::string &/std::string&/`

Comment (weissjeffm) [inline wrappers/opentxs.i]: What?? The formatting checker checks here too? Never would have occurred to me.

**Comment by weissjeffm** [comment]
@monetas-ci re-test this please

**Comment by weissjeffm** [comment]
Fixes #530

**Comment by weissjeffm** [comment]
retest this please

**Comment by weissjeffm** [comment]
retest this please

**Comment by lclc** [comment]
@monetas-ci <https://github.com/monetas-ci> re-test this please

I think you have to write his name

2015-02-13 11:19 GMT-08:00 Jeff Weiss <notifications@github.com>:

**Comment by weissjeffm** [comment]
Nope, works fine without it.

**Comment by weissjeffm** [comment]
retest this please

**Comment by weissjeffm** [comment]
retest this please

**Comment by weissjeffm** [comment]
retest this please

**Comment by weissjeffm** [comment]
test this please

**Comment by weissjeffm** [comment]
@monetas-ci test this please

**Comment by weissjeffm** [comment]
sorry for the spam, fixing jenkins.

retest this please

**Comment by weissjeffm** [comment]
test this please

**Comment by toxeus** [comment]
:+1:

**Pull request merged**